pandas/io/tests/parser/common.py

```diff
@@ -1602,3 +1602,26 @@ def test_internal_eof_byte(self):
         expected = pd.DataFrame([["1\x1a", 2]], columns=['a', 'b'])
         result = self.read_csv(StringIO(data))
         tm.assert_frame_equal(result, expected)
+
+    def test_file_handles(self):
```

Comment:
yes, but I would like to keep those tests together, and only the mmap does not work with the python engine (I don't test the actual reading of a mmap, only that the handler is not closed)